tests/test_airfoi_func.py

```diff
@@ -0,0 +1,75 @@
+import sys
+import pickle
+import os
+import gmshairfoil2d.__init__
+from gmshairfoil2d.airfoil_func import (
+    get_all_available_airfoil_names,
+    get_airfoil_file,
+    NACA_4_digit_geom,
+)
+
+
+LIB_DIR = os.path.dirname(gmshairfoil2d.__init__.__file__)
+database_dir = LIB_DIR + "/../database"
+test_data_dir = LIB_DIR + "/../tests/test_data"
+
+
+def test_get_all_available_airfoil_names():
+    """
+    Test if at least the database obtained containt some airfoils
+    
+    """
+
+    expected_airfoil = ["naca0010", "naca0018", "falcon", "goe510", "e1210"]
+    current_airfoil_list = get_all_available_airfoil_names()
+
+    for foil in expected_airfoil:
+        assert foil in current_airfoil_list, "test passed"
+
```

Comment:
In one line I think it could be:
assert all(airfoil in current_airfoil_list for airfoil in expected_airfoil)

but maybe we should check that a bit differently, we'll see together.